src/run_releasenotes_check.py

```diff
@@ -0,0 +1,94 @@
+        logging.info(f"Checking out into {work_dir.name}")
+        for component in manifest.components.select():
+            logging.info(f"Checking out {component.name}")
+            repo_list = []  # type: List[Any]
+            if type(component) is InputComponentFromSource:
+                with GitRepository(
+                    component.repository,
+                    component.ref,
+                    os.path.join(work_dir.name, component.name),
+                    component.working_directory,
+                ) as repo:
+                    logging.debug(f"Checked out {component.name} into {repo.dir}")
+                    gitLogCmd = f'git -C {repo.dir} log --after={gitLogDate} --pretty=format:"%h"'
+                    gitLog = subprocess.check_output(gitLogCmd.split()).decode()
+                    repo_list.append(component.name)
+                    repo_list.append(component.ref)
+                    repo_list.append(gitLog)
+                    release_notes = "NO"
+                    if component.name == 'OpenSearch' or component.name == 'OpenSearch-Dashboards':
+                        release_notes_check = f'.release-notes-{manifest.build.version}.md'
+                    else:
+                        release_notes_check = f'.release-notes-{manifest.build.version}.0.md'
+                    try:
+                        if any(fname.endswith(release_notes_check) for fname in os.listdir(f'{repo.dir}/release-notes/')):
+                            release_notes = "YES"
+                    except FileNotFoundError:
+                        print("No such release-notes file or directory")
+                        release_notes = "NULL"
+                    repo_list.append(release_notes)
+                    value_matrix.append(repo_list)
+    writer.write_table()
+    writer.dump("table.txt")
+    if addComment == "true":
+        if os.environ.get('GIT_ISSUE_NUMBER') is not None:
+            comment_under_issue(gitIssueNumber, 'table.txt', userToken)
+        else:
+            print("No GIT_ISSUE_NUMBER environmental value passed")
+            exit()
+    subprocess_cmd('rm table.txt')
+    return 0
+
+
+def subprocess_cmd(command: str) -> int:
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
+    proc_stdout = process.communicate()[0].strip()
+    print(proc_stdout)
+    return 0
+
+
+def comment_under_issue(issue_number: str, comment_text: str, userToken: str) -> int:
+    # GITHUB_TOKEN used to add comment, to passed in GH issue, using GIT_ISSUE_NUMBER
+    subprocess_cmd('echo {} | gh auth login --with-token'.format(userToken))
+    subprocess_cmd('gh repo clone https://github.com/opensearch-project/opensearch-build.git; cd opensearch-build')
+    subprocess_cmd('gh issue comment {} --body-file ../{} --repo opensearch-project/opensearch-build'.format(issue_number, comment_text))
+    subprocess_cmd('gh auth logout --hostname github.com; cd ..; rm -rf opensearch-build')
```

Comment:
Remove the `rm -rf` so above temp dir removal can take care of it automatically.

Reply:
Just pushed in my latest commit @peterzhuamazon.